flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/converters/SqlNodeConverters.java

```diff
@@ -53,6 +53,7 @@ public class SqlNodeConverters {
         register(new SqlTruncateTableConverter());
         register(new SqlShowFunctionsConverter());
         register(new SqlShowModelsConverter());
+        register(new SqlShowCreateModelConverter());
```

Comment:
I am curious - are we missing a showCreateTableConverter?

Reply:
It's not using this pattern: https://github.com/apache/flink/blob/master/flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlNodeToOperationConversion.java#L322

The converters pattern is introduced after table operations added. There can be followups to migration table operations to converters maybe